components/library/mozilla-nspr/Makefile

```diff
@@ -73,9 +69,9 @@ COMPONENT_TEST_ENV_CMD=
 COMPONENT_TEST_ARGS=
 COMPONENT_TEST_TARGETS=
 
-test:		$(TEST_32_and_64)
-
+# Build dependencies
 REQUIRED_PACKAGES += text/xmlto
 
 # Auto-generated dependencies
+REQUIRED_PACKAGES += SUNWcs
```

Comment:
Did we not want to deprecate this?

SUNWcs is huge and pulls in system/library and everything else to a total of 1GiB dependencies.

Reply:
This part is auto-generated so there seems to be a dependency yet. I am not sure how to proceed...

Reply:
hmm unless @Mno-hime has an idea how to take it out then leave it in. I thought Auto generated would ignore it. But that discussion was a long time ago. It won't hurt us right now.

Reply:
I don't even remember that discussion :). I always keep it. I doubt we can avoid it in reality?

Reply:
Oh wow that must have been long ago then. Well in that case SUNWcs will be linked into every package and we will have to look into if we want to get rid of it like omnios has. Carry on then. LGTM

Reply:
Not package, just Makefile. REQUIRED_PACKAGE = build dependencies + packages which used to resolve dependencies . It's not a hard-coded list of dependencies.